tensorflow/python/lib/io/file_io.py

```diff
@@ -420,6 +437,21 @@ def get_matching_files_v2(pattern):
             compat.as_bytes(single_filename))
     ]
 
+  def handle_single_pattern(single_pattern):
```

Comment:
I would move this to the beginning of the function and use it in the `if isinstance(pattern, six.string_types):` case too.

Reply:
Have done.